components/raftstore/src/store/config.rs

```diff
@@ -240,6 +245,7 @@ impl Default for Config {
             future_poll_size: 1,
             hibernate_regions: true,
             early_apply: true,
+            ensure_all_target_peer_exist: true,
```

Comment:
Default should be false.

Reply:
It should be `true`. If so, an error log will be printed when something is wrong and breaks this guarantee.

Reply:
How? From what I can see, it just `debug_assert`.

Reply:
I misrepresented it. It should be false only in test just for passing some tests and not panic. `debug_assert` will be replaced to another one.